content/docs/getting-started/quick-start.md

```diff
@@ -7,22 +7,33 @@ aliases:
 
 ## Installing
 
-Let's get started by installing Scratch Addons. Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+Let's get started by installing Scratch Addons.
+
+| Store | Install | Supported browsers |
+| - | - | - |
+| Chrome Web Store | [![Install for Chrome Web Store](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=install&color=4285F4)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco) | Google Chrome and other Chromium-based browser

```

Comment:
> other Chromium-based browser

Should be "others" and "browsers"

Reply:
I'm pretty sure it should be "other" and "browsers"

Reply:
Yeah, my bad